packages/google_maps_flutter/google_maps_flutter_android/lib/src/messages.g.dart

```diff
@@ -1048,50 +1083,56 @@ class _PigeonCodec extends StandardMessageCodec {
     } else if (value is PlatformPolyline) {
       buffer.putUint8(138);
       writeValue(buffer, value.encode());
-    } else if (value is PlatformTile) {
+    } else if (value is PlatformPatternItem) {
       buffer.putUint8(139);
```

Comment:
I assume no but is there an issue with these ints changing values and users having miss matched versions?

Reply:
Can you clarify what you mean by mismatched versions? Specifically, of what? These ints are only used within `google_maps_flutter_android`, at least as far as I can tell, which should mean that as long as they are consistent within each version, this should not pose an issue. Do you think I am mistaken about this, or misunderstanding your concern?

Reply:
> These ints are only used within `google_maps_flutter_android`, at least as far as I can tell

That is correct, we never allow the two sides of Pigeon-generated code to span packages, because it would mean that any Pigeon changes or message change would require breaking changes, which would be a nightmare. We always use it internal to each implementation package.

(In theory someone could accidentally land a PR that only updated one language, but it would be a hard mistake to make, and a very hard mistake to get through automated test. We've discussed having a Pigeon "handshake" protocol to error out if the two sides don't match, but we haven't implemented it since it's low priority.)

Reply:
Thanks that helps me understand the code better.